### PR TITLE
Restore migration history

### PR DESCRIPTION
**Pull request by d-wysocki**
This PR create one migration history line between `2.11` and `3.0`.

- Copied migrations related to adding search vector from `2.11` into `3.0`
- Removed old migrations related to adding search vector from `3.0`

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

File: saleor/product/migrations/0141_merge_20210212_1037.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("product", "0139_description_vector_search"),
+        ("product", "0140_auto_20210125_0905"),
+    ]
```
Comment (patrys) [inline saleor/product/migrations/0141_merge_20210212_1037.py:11]: Why is this migration needed?

File: saleor/product/models.py [hunk 2]
```diff
@@ -311,7 +311,7 @@ class Product(SeoModel, ModelWithMetadata):
     description = SanitizedJSONField(
         blank=True, default=dict, sanitizer=clean_editor_js
     )
-    description_plaintext = TextField(blank=True, default="")
+    description_plaintext = TextField(blank=True)
```
Comment (patrys) [inline saleor/product/models.py:314]: Is this related?

File: saleor/product/migrations/0141_merge_20210212_1037.py (hunk 1 above)
Comment (d-wysocki) [reply saleor/product/migrations/0141_merge_20210212_1037.py:11]: `0139_description_vector_search` is related to 2.11 migrations which are creating search vector, so we need to change it dependency and that change requires merge migration

File: saleor/product/models.py (hunk 2 above)
Comment (d-wysocki) [reply saleor/product/models.py:314]: This is difference between 2.11 and 3.0, i removed it to keep consistency 2.11 and 3.0. This changes doesn't affect on functionality

**Comment by maarcingebala** [comment]
I've tested the branch by running the migrations 2.11 -> 3.0 and setting up fresh 3.0, seems that it works.

File: saleor/product/migrations/0130_create_product_description_search_vector.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+import django.contrib.postgres.indexes
+import django.contrib.postgres.search
+from django.db import migrations, models
+
+
+def parse_draftjs_content_to_string(definitions):
+    string = ""
+    blocks = definitions.get("blocks")
+    if not blocks or not isinstance(blocks, list):
```
Comment (korycins) [inline saleor/product/migrations/0130_create_product_description_search_vector.py:11]:
```
if not isinstance(blocks, list):
```
is enough

File: saleor/product/migrations/0130_create_product_description_search_vector.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+
+def parse_description_json_field(apps, schema):
+    Product = apps.get_model("product", "Product")
+
+    for product in Product.objects.iterator():
+        product.description_plaintext = parse_draftjs_content_to_string(
+            product.description_json
+        )
+        product.save()
```
Comment (korycins) [inline saleor/product/migrations/0130_create_product_description_search_vector.py:29]: `save(update_fields=["description_plaintext"])`

**Review by NyanKiyoshi — CHANGES_REQUESTED** [review]
Reviewing

File: saleor/product/migrations/0130_create_product_description_search_vector.py (hunk 3 above)
Comment (d-wysocki) [reply saleor/product/migrations/0130_create_product_description_search_vector.py:11]: It's code from 2.11, I would like to keep it the same at it was on 2.11

File: saleor/product/migrations/0130_create_product_description_search_vector.py (hunk 4 above)
Comment (d-wysocki) [reply saleor/product/migrations/0130_create_product_description_search_vector.py:29]: It's code from 2.11, I would like to keep it the same at it was on 2.11

**Review by NyanKiyoshi — APPROVED** [review]
I give green light but let's wait on @patrys to give his too

**Review by patrys — APPROVED** [review]
